include/boost/histogram/python/accumulators/ostream.hpp

```diff
@@ -8,21 +8,23 @@
 
 #pragma once
 
+#include <boost/histogram/accumulators/sum.hpp>
+#include <boost/histogram/python/accumulators/mean.hpp>
+#include <boost/histogram/python/accumulators/weighted_mean.hpp>
+#include <boost/histogram/python/accumulators/weighted_sum.hpp>
+
 #include <boost/histogram/detail/counting_streambuf.hpp>
 #include <boost/histogram/fwd.hpp>
 #include <iosfwd>
 
-namespace boost {
-namespace histogram {
-
-namespace detail {
+namespace {
```

Comment:
Anonymous namespace may not be used in headers, see https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file

Reply:
Strictly speaking it is ok if the header is not included in several cxx files, but this is the case here, I think. Also, it is good to embrace such rules instead of thinking about them case by case...

Reply:
> Also, it is good to embrace such rules instead of thinking about them case by case.

Sounds good.